### ISSUE TITLE
Basic setup with create-react-app does not display markdown documentation

### ISSUE THREAD

**Issue by ThomasRoest** [styleguidist/react-styleguidist#1993]
**Current behavior**

This very basic setup with create-react-app does not seem to work as expected. The component is found, but the markdown file isn't? 
In the chrome devtools I do get the following HMR / webpack dev server message: 

`[HMR] Waiting for update signal from WDS...`

**To reproduce**

The repo with example: https://github.com/ThomasRoest/styleguidist-test

**Expected behavior**

By default, it should also find and render the markdown examples.

**Comment by elciocf** [comment]
same here

**Comment by ThomasRoest** [comment]
This can be fixed with the workaround described in https://github.com/styleguidist/react-styleguidist/issues/1910#issuecomment-1013763698

But this still leaves the problem of hot module reloading not working

**Comment by eya26** [comment]
I reproduced the example in the repo with the same versions, however, the markdown examples are rendered.
![styleguidistTest](https://user-images.githubusercontent.com/77161925/164945548-b2776528-fff8-4291-bcaa-808f0f869989.PNG)

![styleguidistTest2](https://user-images.githubusercontent.com/77161925/164945553-bcd9fc41-a18d-4cf8-8e2e-d2754001a255.PNG)

**Comment by ThomasRoest** [comment]
@eya26 yes because it includes this workaround https://github.com/ThomasRoest/styleguidist-test/commit/f11e7e05c1ae68cd6ced924fccdcd6b5915e23cd

**Comment by adubovsky** [comment]
> 

For all who curious, after a few days of debugging, the following config (added to the workaround mentioned above) fixes hot reloading:

```
const webpackConfig = {
  devServer: {
      // Not really necessary, but to match the new webpack defaults
      sockPath: '/ws',
      // Web socket client is not injected by CRA webpack config, so ask webpack
      // to include the default one for us
      injectClient: true,
    },
    module: {
      ...
    },
    ...
}
```
